ddp-workspace/projects/ddp-dsm-ui/src/app/participant-list/participant-list.component.ts

```diff
@@ -2127,7 +2127,7 @@ export class ParticipantListComponent implements OnInit {
   private createFilter(field: any): Filter {
     let showType = field.displayType;
     let filter: Filter = new Filter(
-      new ParticipantColumn(field.columnDisplay.replace('*', ''), field.columnName, 'participantData', field.fieldType, false),
+      new ParticipantColumn(field.columnDisplay?.replace('*', ''), field.columnName, 'participantData', field.fieldType, false),
```

Comment:
Not sure where that `?` came from… @GiCharkviani is that ok here or will that break things?

Reply:
I have added that sign, just for any case, if columnDisplay appears undefined, not to crush the app. That is called ["Optional Chaining"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)